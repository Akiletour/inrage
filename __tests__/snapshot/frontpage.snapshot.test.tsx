import { render } from '@testing-library/react';

import Frontpage from '../../pages/index';

it('renders frontpage without changing', () => {
  const lastProjects = {
    edges: [
      {
        node: {
          id: 1,
          title: 'string',
          slug: 'string',
          status: 'private',
          featuredImage: {
            node: {
              sourceUrl: 'string',
            },
          },
          supports: {
            edges: [
              {
                node: {
                  name: 'string',
                  slug: 'string',
                },
              },
            ],
          },
        },
      },
    ],
  };

  const articles = {
    edges: [
      {
        node: {
          title: 'Hello',
          slug: 'hello',
          excerpt: 'publish',
          date: '2020-01-01 00:00:00',
          featuredImage: { node: { sourceUrl: 'http://' } },
        },
      },
    ],
  };
  const { container } = render(
    <Frontpage lastProjects={lastProjects} articles={articles} />
  );
  expect(container).toMatchSnapshot();
});
