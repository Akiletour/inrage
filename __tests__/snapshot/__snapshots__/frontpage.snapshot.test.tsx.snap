// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders frontpage without changing 1`] = `
<div>
  <div>
    <main
      style="opacity: 0; transform: translateX(-100px) translateY(0px) translateZ(0);"
    >
      <div
        class="container"
      >
        <div
          class="flex md:items-center flex-col md:flex-row mt-3 md:mt-0"
        >
          <h2
            class="false text-white flex-none text-3xl font-bold max-w-[390px]"
          >
            A propos
          </h2>
          <p
            class="mt-2 md:mt-0 text-gray-light flex-1 flex items-center"
          >
            <span
              class="hidden md:block w-[2px] h-[120px] mx-8 bg-gray flex-none -skew-x-[30deg]"
            />
            Développeur Freelance créatif et innovant basé au coeur de La Rochelle et spécialisé dans la création de site internet. Depuis l'âge de 14 ans, je développe, compose et crée des projets web.
          </p>
        </div>
        <div
          class="my-4 mx-auto text-xl sm:text-2xl max-w-4xl font-medium text-center text-white"
        >
          Je suis entouré d'une équipe fondée de créatifs, designers et développeurs. Nous travaillons ensemble pour créer des choses inspirantes et engagées.
        </div>
        <div
          class="grid md:grid-cols-3 gap-6 md:gap-3 lg:gap-6"
        >
          <div
            class="text-center h-full flex flex-col items-center"
          >
            <h3
              class="text-xl text-white font-medium mt-1 leading-7"
            >
              <span
                class="block"
              >
                Conception / Refonte
              </span>
              <span
                class="block"
              >
                de sites Internet
              </span>
            </h3>
            <p
              class="text-base md:text-sm lg:text-base leading-5 lg:leading-5 mt-1 mb-2"
            >
              Sites vitrines, associatifs, e-commerce ou dédiés. Je vous accompagne dès la genèse de votre projet pour vous proposer des solutions puissantes et performantes et répondre le plus efficacement aux enjeux de votre activité. Je suis expert dans la création de site sur les CMS Joomla, WordPress et Prestashop.
            </p>
            <div
              class="mt-auto"
            >
              <a
                href="/prestations/creation-site-web"
              >
                <span
                  class="button"
                >
                  Découvrir mes prestations
                </span>
              </a>
            </div>
          </div>
          <div
            class="text-center h-full flex flex-col items-center"
          >
            <h3
              class="text-xl text-white font-medium mt-1 leading-7"
            >
              <span
                class="block"
              >
                Maintenance &
              </span>
              <span
                class="block"
              >
                Infogérance
              </span>
            </h3>
            <p
              class="text-base md:text-sm lg:text-base leading-5 lg:leading-5 mt-1 mb-2"
            >
              La maintenance d’un site internet s’avère complexe lorsqu’on ne dispose pas du bagage technique nécessaire voire le temps de s’en occuper. Pour ces tâches délicates et chronophages, je vous propose de prendre le relai et d’assurer pour vous la gestion de vos sauvegardes, ainsi que la maintenance et la sécurité de votre serveur dans sa globalité.
            </p>
            <div
              class="mt-auto"
            >
              <a
                href="/prestations/maintenance-site-internet"
              >
                <span
                  class="button"
                >
                  Mes offres de maintenance
                </span>
              </a>
            </div>
          </div>
          <div
            class="text-center h-full flex flex-col items-center"
          >
            <h3
              class="text-xl text-white font-medium mt-1 leading-7"
            >
              <span
                class="block"
              >
                Hébergement optimisé &
              </span>
              <span
                class="block"
              >
                ultra rapide
              </span>
            </h3>
            <p
              class="text-base md:text-sm lg:text-base leading-5 lg:leading-5 mt-1 mb-2"
            >
              Lorsque vous souhaitez mettre en ligne un site, il est nécessaire de choisir une offre d’hébergement parmis une foule d’offres et de prestataires. Nos offres d’hébergement sont transparentes et sans frais cachés. Un expert de notre partenaire Dutiko analyse votre projet ainsi que votre demande pour vous proposer l’offre idéale pour la nature de votre site web.
            </p>
            <div
              class="mt-auto"
            >
              <a
                href="/contact"
              >
                <span
                  class="button"
                >
                  Me contacter
                </span>
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        class="relative"
      >
        <div
          class="relative"
        >
          <svg
            class="-z-10 h-10 sm:h-16 md:h-25 lg:h-36 xl:h-45 w-full block -scale-x-100 -scale-y-100"
            height="361"
            preserveAspectRatio="xMidYMax slice"
            viewBox="0 0 1920 361"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              class="fill-gray-dark"
              d="M1349 234L1920 0v361H0z"
              fill="#000"
            />
            <path
              class="fill-orange"
              d="M1445 197.5L1920 4v249z"
              fill="#BE4C4C"
            />
          </svg>
        </div>
        <div
          class="container relative z-10 py-5 xl:py-0 md:-my-10"
        >
          <div
            class="md:w-1/2"
          >
            <h2
              class="text-2xl sm:text-3xl text-white font-medium mb-3"
            >
              Besoin d’une estimation de votre site
              <br
                class="hidden sm:block"
              />
              et/ou de sa maintenance
            </h2>
            <p>
              Simple et en toute transparence, n’hésitez pas à me demander une estimation gratuite pour la création de votre site web ou de la maintenance applicative.
            </p>
            <p>
              Vous recevrez dans la journée votre devis et nous pourrons prendre contact pour définir ensemble les tâches.
            </p>
            <div
              class="flex flex-wrap flex-start"
            >
              <a
                href="/contact"
              >
                <span
                  class="button mt-3 mr-2"
                >
                  Demandez votre devis maintenant
                </span>
              </a>
              <a
                href="/prestations/maintenance-site-internet"
              >
                <span
                  class="button-outline mt-3"
                >
                  Mes offres de maintenance
                </span>
              </a>
            </div>
          </div>
        </div>
        <div
          class="relative"
        >
          <svg
            class="h-10 sm:h-16 md:h-25 lg:h-36 xl:h-45 w-full block"
            height="361"
            preserveAspectRatio="xMidYMax slice"
            viewBox="0 0 1920 361"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              class="fill-gray-darker"
              d="M1349 234L1920 0v361H0z"
              fill="#000"
            />
            <path
              class="fill-orange"
              d="M1445 197.5L1920 4v249z"
              fill="#BE4C4C"
            />
          </svg>
        </div>
      </div>
      <div
        class="bg-gray-darker pt-8 md:pt-3"
      >
        <div
          class="container mb-10 lg:-mb-8 z-10 relative"
        >
          <div
            class="flex md:items-center flex-col md:flex-row "
          >
            <h2
              class="false text-white flex-none text-3xl font-bold max-w-[390px]"
            >
              Projets
            </h2>
            <p
              class="mt-2 md:mt-0 text-gray-light flex-1 flex items-center"
            >
              <span
                class="hidden md:block w-[2px] h-[120px] mx-8 bg-gray flex-none -skew-x-[30deg]"
              />
              Consultez mes dernières créations, atteignant tous, l’esthétique du détail et de la fonctionnalité qui me démarque du reste en tant que développeur Freelance.
            </p>
          </div>
          <div
            class="mt-3 sm:mt-0 grid gap-2 sm:gap-0 grid-cols-2 md:grid-cols-4"
          >
            <a
              href="/portfolio/string/string"
            >
              <div
                class="project-item group text-center"
              >
                <div
                  class="md:group-hover:scale-110 relative transition-all block text-center sm:pt-[5%] sm:px-[10%] sm:pb-[7%]"
                />
                <div
                  class="text-white text-base leading-5 sm:text-2xl"
                >
                  string
                </div>
                <div
                  class="text-xs mt-1 sm:text-base md:text-sm lg:text-base uppercase"
                >
                  string
                </div>
              </div>
            </a>
          </div>
        </div>
        <div
          class="relative"
        >
          <svg
            class="h-10 sm:h-16 md:h-25 lg:h-36 xl:h-45 w-full block"
            height="361"
            preserveAspectRatio="xMidYMax slice"
            viewBox="0 0 1920 361"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              class="fill-gray-dark"
              d="M1349 234L1920 0v361H0z"
              fill="#000"
            />
            <path
              class="fill-orange"
              d="M1445 197.5L1920 4v249z"
              fill="#BE4C4C"
            />
          </svg>
          <a
            href="/portfolio"
          >
            <div
              class="absolute left-1/2 -bottom-2 md:bottom-1 -translate-x-1/2 w-14 md:w-18 h-14 md:h-18 bg-gray-dark text-gray hover:text-white rounded-full flex flex-col items-center pt-3"
            >
              <div>
                <svg
                  class=""
                  height="24"
                  viewBox="0 0 512 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"
                    fill="currentColor"
                  />
                </svg>
              </div>
              <div
                class="mt-1 text-xs md:text-sm text-center leading-4"
              >
                <span
                  class="block"
                >
                  Voir tous
                </span>
                <span
                  class="block"
                >
                  les projets
                </span>
              </div>
            </div>
          </a>
        </div>
      </div>
      <div
        class="container"
      >
        <div
          class="flex md:items-center flex-col md:flex-row "
        >
          <h2
            class="false text-white flex-none text-3xl font-bold max-w-[390px]"
          >
            Expertises
          </h2>
          <p
            class="mt-2 md:mt-0 text-gray-light flex-1 flex items-center"
          >
            <span
              class="hidden md:block w-[2px] h-[120px] mx-8 bg-gray flex-none -skew-x-[30deg]"
            />
            Je propose un large éventail de services axés sur les résultats pour les marques, en veillant à ce que leur présence en ligne reflète réellement leurs objectifs et leurs inspirations.
          </p>
        </div>
        <div
          class="my-4 grid md:grid-cols-2 gap-x-2 gap-y-6"
        >
          <div
            class="grid gap-x-3 grid-cols-[80px_auto] md:grid-cols-[120px_auto] items-center"
          >
            <div
              class="relative mr-2 w-10 h-10 md:w-15 md:h-15 row-span-2"
            />
            <h3
              class="text-white font-medium text-xl md:text-2xl"
            >
              WordPress
            </h3>
            <div
              class="col-auto"
            >
              <p
                class="mb-1 text-sm sm:text-md md:text-base"
              >
                WordPress est le plus populaire des CMS en ce moment, il vous permet d’administrer facilement votre site et de personnaliser intégralement le frontoffice.
              </p>
              <a
                href="/prestations/creation-site-web/site-wordpress"
              >
                <span
                  class="group hover:text-orange transition-all text-white font-medium flex items-center"
                >
                  En savoir plus
                  <svg
                    class="ml-2 h-3 group-hover:animate-bouncex"
                    viewBox="0 0 448 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      class="fill-orange"
                      d="M313.941 216H12c-6.627 0-12 5.373-12 12v56c0 6.627 5.373 12 12 12h301.941v46.059c0 21.382 25.851 32.09 40.971 16.971l86.059-86.059c9.373-9.373 9.373-24.569 0-33.941l-86.059-86.059c-15.119-15.119-40.971-4.411-40.971 16.971V216z"
                      fill="currentColor"
                    />
                  </svg>
                </span>
              </a>
            </div>
          </div>
          <div
            class="grid gap-x-3 grid-cols-[80px_auto] md:grid-cols-[120px_auto] items-center"
          >
            <div
              class="relative mr-2 w-10 h-10 md:w-15 md:h-15 row-span-2"
            />
            <h3
              class="text-white font-medium text-xl md:text-2xl"
            >
              Symfony
            </h3>
            <div
              class="col-auto"
            >
              <p
                class="mb-1 text-sm sm:text-md md:text-base"
              >
                Symfony est un framework PHP qui nous permet d’accélèrer le développement de sites ou d’applications grace à sa méthodologie et architecture évolutive.
              </p>
              <a
                href="/prestations/creation-site-web/symfony"
              >
                <span
                  class="group hover:text-orange transition-all text-white font-medium flex items-center"
                >
                  En savoir plus
                  <svg
                    class="ml-2 h-3 group-hover:animate-bouncex"
                    viewBox="0 0 448 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      class="fill-orange"
                      d="M313.941 216H12c-6.627 0-12 5.373-12 12v56c0 6.627 5.373 12 12 12h301.941v46.059c0 21.382 25.851 32.09 40.971 16.971l86.059-86.059c9.373-9.373 9.373-24.569 0-33.941l-86.059-86.059c-15.119-15.119-40.971-4.411-40.971 16.971V216z"
                      fill="currentColor"
                    />
                  </svg>
                </span>
              </a>
            </div>
          </div>
          <div
            class="grid gap-x-3 grid-cols-[80px_auto] md:grid-cols-[120px_auto] items-center"
          >
            <div
              class="relative mr-2 w-10 h-10 md:w-15 md:h-15 row-span-2"
            />
            <h3
              class="text-white font-medium text-xl md:text-2xl"
            >
              Prestashop
            </h3>
            <div
              class="col-auto"
            >
              <p
                class="mb-1 text-sm sm:text-md md:text-base"
              >
                Prestashop est un CMS très puissant pour les sites de commerce électronique (e-commerce) avec plus de 5 000 modules et thèmes.
              </p>
              <a
                href="/prestations/creation-site-web/site-prestashop"
              >
                <span
                  class="group hover:text-orange transition-all text-white font-medium flex items-center"
                >
                  En savoir plus
                  <svg
                    class="ml-2 h-3 group-hover:animate-bouncex"
                    viewBox="0 0 448 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      class="fill-orange"
                      d="M313.941 216H12c-6.627 0-12 5.373-12 12v56c0 6.627 5.373 12 12 12h301.941v46.059c0 21.382 25.851 32.09 40.971 16.971l86.059-86.059c9.373-9.373 9.373-24.569 0-33.941l-86.059-86.059c-15.119-15.119-40.971-4.411-40.971 16.971V216z"
                      fill="currentColor"
                    />
                  </svg>
                </span>
              </a>
            </div>
          </div>
          <div
            class="grid gap-x-3 grid-cols-[80px_auto] md:grid-cols-[120px_auto] items-center"
          >
            <div
              class="relative mr-2 w-10 h-10 md:w-15 md:h-15 row-span-2"
            />
            <h3
              class="text-white font-medium text-xl md:text-2xl"
            >
              Joomla
            </h3>
            <div
              class="col-auto"
            >
              <p
                class="mb-1 text-sm sm:text-md md:text-base"
              >
                Joomla est un CMS puissant en terme de pérénité, modularité et de puissance dans la gestion de contenu et de l’ajout de vos développement spécifique.
              </p>
              <a
                href="/portfolio"
              >
                <span
                  class="group hover:text-orange transition-all text-white font-medium flex items-center"
                >
                  En savoir plus
                  <svg
                    class="ml-2 h-3 group-hover:animate-bouncex"
                    viewBox="0 0 448 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      class="fill-orange"
                      d="M313.941 216H12c-6.627 0-12 5.373-12 12v56c0 6.627 5.373 12 12 12h301.941v46.059c0 21.382 25.851 32.09 40.971 16.971l86.059-86.059c9.373-9.373 9.373-24.569 0-33.941l-86.059-86.059c-15.119-15.119-40.971-4.411-40.971 16.971V216z"
                      fill="currentColor"
                    />
                  </svg>
                </span>
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        class="relative"
      >
        <div
          class="relative"
        >
          <svg
            class="h-10 sm:h-20 lg:h-auto w-full block -scale-x-100 -scale-y-100"
            height="361"
            preserveAspectRatio="xMidYMax slice"
            viewBox="0 0 1920 361"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              class="fill-gray-dark"
              d="M1349 234L1920 0v361H0z"
              fill="#000"
            />
            <path
              class="fill-orange"
              d="M1445 197.5L1920 4v249z"
              fill="#BE4C4C"
            />
          </svg>
        </div>
        <div
          class="container grid grid-cols-2 md:grid-cols-4 gap-y-4 py-4"
        >
          <div
            class="text-center text-white flex items-center flex-col"
          >
            <div
              class="w-10 md:w-15 h-10 md:h-15 border-2 border-white rounded-full flex items-center justify-center"
            >
              <svg
                class="h-3 md:h-5"
                height="24"
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M127.1 146.5c1.3 7.7 8 13.5 16 13.5h16.5c9.8 0 17.6-8.5 16.3-18-3.8-28.2-16.4-54.2-36.6-74.7-14.4-14.7-23.6-33.3-26.4-53.5C111.8 5.9 105 0 96.8 0H80.4C70.6 0 63 8.5 64.1 18c3.9 31.9 18 61.3 40.6 84.4 12 12.2 19.7 27.5 22.4 44.1zm112 0c1.3 7.7 8 13.5 16 13.5h16.5c9.8 0 17.6-8.5 16.3-18-3.8-28.2-16.4-54.2-36.6-74.7-14.4-14.7-23.6-33.3-26.4-53.5C223.8 5.9 217 0 208.8 0h-16.4c-9.8 0-17.5 8.5-16.3 18 3.9 31.9 18 61.3 40.6 84.4 12 12.2 19.7 27.5 22.4 44.1zM400 192H32c-17.7 0-32 14.3-32 32v192c0 53 43 96 96 96h192c53 0 96-43 96-96h16c61.8 0 112-50.2 112-112s-50.2-112-112-112zm0 160h-16v-96h16c26.5 0 48 21.5 48 48s-21.5 48-48 48z"
                  fill="currentColor"
                />
              </svg>
            </div>
            <div
              class="font-medium text-2xl mt-2"
            >
              2356
            </div>
            <div
              class="leading-5"
            >
              tasses à café
            </div>
          </div>
          <div
            class="text-center text-white flex items-center flex-col"
          >
            <div
              class="w-10 md:w-15 h-10 md:h-15 border-2 border-white rounded-full flex items-center justify-center"
            >
              <svg
                class="h-3 md:h-5"
                height="25"
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M104 224H24c-13.255 0-24 10.745-24 24v240c0 13.255 10.745 24 24 24h80c13.255 0 24-10.745 24-24V248c0-13.255-10.745-24-24-24zM64 472c-13.255 0-24-10.745-24-24s10.745-24 24-24 24 10.745 24 24-10.745 24-24 24zM384 81.452c0 42.416-25.97 66.208-33.277 94.548h101.723c33.397 0 59.397 27.746 59.553 58.098.084 17.938-7.546 37.249-19.439 49.197l-.11.11c9.836 23.337 8.237 56.037-9.308 79.469 8.681 25.895-.069 57.704-16.382 74.757 4.298 17.598 2.244 32.575-6.148 44.632C440.202 511.587 389.616 512 346.839 512l-2.845-.001c-48.287-.017-87.806-17.598-119.56-31.725-15.957-7.099-36.821-15.887-52.651-16.178-6.54-.12-11.783-5.457-11.783-11.998v-213.77c0-3.2 1.282-6.271 3.558-8.521 39.614-39.144 56.648-80.587 89.117-113.111 14.804-14.832 20.188-37.236 25.393-58.902C282.515 39.293 291.817 0 312 0c24 0 72 8 72 81.452z"
                  fill="currentColor"
                />
              </svg>
            </div>
            <div
              class="font-medium text-2xl mt-2"
            >
              352
            </div>
            <div
              class="leading-5"
            >
              projets
            </div>
          </div>
          <div
            class="text-center text-white flex items-center flex-col"
          >
            <div
              class="w-10 md:w-15 h-10 md:h-15 border-2 border-white rounded-full flex items-center justify-center"
            >
              <svg
                class="h-3 md:h-5"
                viewBox="0 0 576 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M552 64H448V24c0-13.3-10.7-24-24-24H152c-13.3 0-24 10.7-24 24v40H24C10.7 64 0 74.7 0 88v56c0 66.5 77.9 131.7 171.9 142.4C203.3 338.5 240 360 240 360v72h-48c-35.3 0-64 20.7-64 56v12c0 6.6 5.4 12 12 12h296c6.6 0 12-5.4 12-12v-12c0-35.3-28.7-56-64-56h-48v-72s36.7-21.5 68.1-73.6C498.4 275.6 576 210.3 576 144V88c0-13.3-10.7-24-24-24zM64 144v-16h64.2c1 32.6 5.8 61.2 12.8 86.2-47.5-16.4-77-49.9-77-70.2zm448 0c0 20.2-29.4 53.8-77 70.2 7-25 11.8-53.6 12.8-86.2H512v16zm-127.3 4.7-39.6 38.6 9.4 54.6c1.7 9.8-8.7 17.2-17.4 12.6l-49-25.8-49 25.8c-8.8 4.6-19.1-2.9-17.4-12.6l9.4-54.6-39.6-38.6c-7.1-6.9-3.2-19 6.7-20.5l54.8-8 24.5-49.6c4.4-8.9 17.1-8.9 21.5 0l24.5 49.6 54.8 8c9.6 1.5 13.5 13.6 6.4 20.5z"
                  fill="currentColor"
                />
              </svg>
            </div>
            <div
              class="font-medium text-2xl mt-2"
            >
              15 années
            </div>
            <div
              class="leading-5"
            >
              d'expérience
            </div>
          </div>
          <div
            class="text-center text-white flex items-center flex-col"
          >
            <div
              class="w-10 md:w-15 h-10 md:h-15 border-2 border-white rounded-full flex items-center justify-center"
            >
              <svg
                class="h-3 md:h-5"
                height="24"
                viewBox="0 0 576 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M546.2 9.7c-5.6-12.5-21.6-13-28.3-1.2C486.9 62.4 431.4 96 368 96h-80C182 96 96 182 96 288c0 19.2 2.9 37.7 8.1 55.1C31.6 399.5 4.3 463.2 2.4 468c-6.6 16.3 1.2 34.9 17.5 41.6 16.4 6.8 35-1.1 41.8-17.3 1.5-3.6 20.9-47.9 71.9-90.6 32.4 43.9 94 85.8 174.9 77.2C465.5 467.5 576 326.7 576 154.3c0-50.2-10.8-102.2-29.8-144.6zM430.6 294.4 345.9 380c-5.5 5.5-14.3 5.5-19.8 0l-84.7-85.6c-24.6-24.9-23.2-66.1 4.3-89.1 24-20 59.6-16.4 81.6 5.8l8.6 8.7 8.6-8.7c22-22.2 57.7-25.8 81.6-5.8 27.6 22.9 29.1 64.2 4.5 89.1z"
                  fill="currentColor"
                />
              </svg>
            </div>
            <div
              class="font-medium text-2xl mt-2"
            >
              150+
            </div>
            <div
              class="leading-5"
            >
              idées à développer
            </div>
          </div>
        </div>
        <div
          class="relative"
        >
          <svg
            class="h-10 sm:h-20 lg:h-auto w-full block"
            height="361"
            preserveAspectRatio="xMidYMax slice"
            viewBox="0 0 1920 361"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              class="fill-gray-darker"
              d="M1349 234L1920 0v361H0z"
              fill="#000"
            />
            <path
              class="fill-orange"
              d="M1445 197.5L1920 4v249z"
              fill="#BE4C4C"
            />
          </svg>
        </div>
      </div>
      <div
        class="bg-gray-darker"
      >
        <div
          class="container py-4"
        >
          <div
            class="flex md:items-center flex-col md:flex-row "
          >
            <h2
              class="false text-white flex-none text-3xl font-bold max-w-[390px]"
            >
              Articles
            </h2>
            <p
              class="mt-2 md:mt-0 text-gray-light flex-1 flex items-center"
            >
              <span
                class="hidden md:block w-[2px] h-[120px] mx-8 bg-gray flex-none -skew-x-[30deg]"
              />
              Passionné par les nouvelles technologies, j'adore partager mes compétences et mes découvertes avec des personnes qui ont cette même passion pour le web !
            </p>
          </div>
          <div
            class="grid md:grid-cols-2 gap-4 mb-8 lg:-mb-8 mt-6"
          >
            <div>
              <div
                class="relative flex flex-col"
              >
                <div
                  class="absolute top-1 w-6 text-center left-1 z-10 bg-white"
                >
                  <div
                    class="py-1 bg-orange font-medium text-gray-darker text-2xl"
                  >
                    01
                  </div>
                  <div
                    class="text-gray-darker uppercase"
                  >
                    jan
                  </div>
                </div>
                <a
                  href="/blog/hello"
                >
                  <span />
                </a>
                <h3
                  class="text-white text-2xl font-medium mt-1"
                >
                  Hello
                </h3>
                <div
                  class="mt-1 mb-2"
                >
                  publish
                </div>
                <a
                  href="/blog/hello"
                >
                  <span
                    class="group hover:text-orange transition-all text-white font-medium flex items-center"
                  >
                    Lire la suite
                    <svg
                      class="ml-2 h-3 group-hover:animate-bouncex"
                      viewBox="0 0 448 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        class="fill-orange"
                        d="M313.941 216H12c-6.627 0-12 5.373-12 12v56c0 6.627 5.373 12 12 12h301.941v46.059c0 21.382 25.851 32.09 40.971 16.971l86.059-86.059c9.373-9.373 9.373-24.569 0-33.941l-86.059-86.059c-15.119-15.119-40.971-4.411-40.971 16.971V216z"
                        fill="currentColor"
                      />
                    </svg>
                  </span>
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="relative"
        >
          <svg
            class="h-10 sm:h-16 md:h-25 lg:h-36 xl:h-45 w-full block"
            height="361"
            preserveAspectRatio="xMidYMax slice"
            viewBox="0 0 1920 361"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              class="fill-gray-dark"
              d="M1349 234L1920 0v361H0z"
              fill="#000"
            />
            <path
              class="fill-orange"
              d="M1445 197.5L1920 4v249z"
              fill="#BE4C4C"
            />
          </svg>
          <a
            href="/blog"
          >
            <div
              class="absolute left-1/2 -bottom-2 md:bottom-1 -translate-x-1/2 w-14 md:w-18 h-14 md:h-18 bg-gray-dark text-gray hover:text-white rounded-full flex flex-col items-center pt-3"
            >
              <div>
                <svg
                  class=""
                  height="24"
                  viewBox="0 0 576 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M546.2 9.7c-5.6-12.5-21.6-13-28.3-1.2C486.9 62.4 431.4 96 368 96h-80C182 96 96 182 96 288c0 19.2 2.9 37.7 8.1 55.1C31.6 399.5 4.3 463.2 2.4 468c-6.6 16.3 1.2 34.9 17.5 41.6 16.4 6.8 35-1.1 41.8-17.3 1.5-3.6 20.9-47.9 71.9-90.6 32.4 43.9 94 85.8 174.9 77.2C465.5 467.5 576 326.7 576 154.3c0-50.2-10.8-102.2-29.8-144.6zM430.6 294.4 345.9 380c-5.5 5.5-14.3 5.5-19.8 0l-84.7-85.6c-24.6-24.9-23.2-66.1 4.3-89.1 24-20 59.6-16.4 81.6 5.8l8.6 8.7 8.6-8.7c22-22.2 57.7-25.8 81.6-5.8 27.6 22.9 29.1 64.2 4.5 89.1z"
                    fill="currentColor"
                  />
                </svg>
              </div>
              <div
                class="mt-1 text-xs md:text-sm text-center leading-4"
              >
                <span
                  class="block"
                >
                  Voir tous
                </span>
                <span
                  class="block"
                >
                  les articles
                </span>
              </div>
            </div>
          </a>
        </div>
      </div>
      <div
        class="container mt-8 md:mt-0"
      >
        <div
          class="flex md:items-center flex-col md:flex-row "
        >
          <h2
            class="false text-white flex-none text-3xl font-bold max-w-[390px]"
          >
            Contact
          </h2>
          <p
            class="mt-2 md:mt-0 text-gray-light flex-1 flex items-center"
          >
            <span
              class="hidden md:block w-[2px] h-[120px] mx-8 bg-gray flex-none -skew-x-[30deg]"
            />
            Pour toute demande ou devis, n'hésitez pas à me contacter en remplissant le formulaire ci-dessous, je serais ravis de vous répondre.
          </p>
        </div>
        <form
          class="my-4"
        >
          <div
            class="grid gap-3 md:grid-cols-2"
          >
            <div
              class="relative order-1"
            >
              <input
                class="input-field border-gray"
                name="name"
                placeholder="Prénom NOM"
                type="text"
              />
            </div>
            <div
              class="relative order-3"
            >
              <input
                class="input-field border-gray"
                name="email"
                placeholder="Votre adresse e-mail"
                type="email"
              />
            </div>
            <div
              class="relative order-4"
            >
              <input
                class="input-field border-gray"
                name="phone"
                placeholder="Numéro de téléphone"
                type="tel"
              />
            </div>
            <div
              class="md:row-span-3 order-10 md:order-2 relative"
            >
              <textarea
                class="input-field min-h-[200px] h-full border-gray"
                name="content"
                placeholder="Laissez-moi un petit message"
              />
            </div>
          </div>
          <div
            class="mt-3 flex flex-col md:flex-row items-center justify-center md:justify-end"
          >
            <button
              class="button relative button py-3 px-4 w-full md:w-auto"
              type="submit"
            >
              Envoyer mon message
            </button>
          </div>
        </form>
      </div>
    </main>
  </div>
</div>
`;
